mixer/pkg/protobuf/yaml/dynamic/handler.go

```diff
@@ -379,16 +402,25 @@ func (s Svc) encodeRequest(qr proto.Marshaler, dedupID string, encodedInstances
 }
 
 // Marshal does a noo-op marshal if input in bytes, otherwise it is an error.
-func (Codec) Marshal(v interface{}) ([]byte, error) {
+func (c *Codec) Marshal(v interface{}) ([]byte, error) {
 	if ba, ok := v.([]byte); ok {
 		return ba, nil
 	}
 
 	return nil, fmt.Errorf("unable to marshal type:%T, want []byte", v)
 }
 
-// Unmarshal delegates to standard proto Unmarshal
-func (Codec) Unmarshal(data []byte, v interface{}) error {
+// Unmarshal uses custom decoder or delegates to standard proto Unmarshal
+func (c *Codec) Unmarshal(data []byte, v interface{}) error {
```

Comment:
why `* Codec` instead of `Code`

Reply:
Done. Changed to struct since it's a small value.